tests/Module/CheckoutTest.php

```diff
@@ -568,6 +569,7 @@ private function createCheckoutMock(
 			$carrierEntityRepository,
 			$this->createMock( Api\Internal\CheckoutRouter::class ),
 			$carDeliveryConfig,
+			$this->createMock( PaymentHelper::class ),
```

Comment:
to je tu proč? do checkout třídy se nešahalo :)

Reply:
V mainu neprocházejí testy... případně dám pryč.

Reply:
stačí si rebasnout na novější verzi, mělo by být opraveno, že?